codestyle/checkstyle.xml

```diff
@@ -319,5 +319,11 @@ codestyle/checkstyle.xml.&#10;"/>
     <module name="LocalFinalVariableName">
       <property name="format" value="^[a-z_]*[a-z0-9][a-zA-Z0-9_]*$"/>
     </module>
+
+    <module name="Regexp">
```

Comment:
Could you use [forbidden-apis](https://github.com/apache/incubator-druid/blob/master/codestyle/druid-forbidden-apis.txt) which is a native tool for this type of prohibition?

Reply:
Oops, yeah will fix :+1: